### PR TITLE
Send some messages to syslog

### PR DESCRIPTION
**Pull request by TuxPowered42**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -167,12 +192,20 @@ func (s *Appender) archiveFileName() string {
 	return s.filePath + "_" + ts
 }
 
+// Append inserts line at the end of file and asks file to be rotated if it is too big.
 func (s *Appender) Append(line string) {
 	if s.bytesWritten >= *maxFileSize {
 		s.rotateFile()
 	}
 	n, _ := s.writer.WriteString(line)
+	if s.syslog != nil && s.regexp != nil {
```
Comment (Woellchen) [inline main.go]: why are you putting the syslog part between the file log part? move it down please

File: main.go [hunk 2]
```diff
@@ -167,12 +192,20 @@ func (s *Appender) archiveFileName() string {
 	return s.filePath + "_" + ts
 }
 
+// Append inserts line at the end of file and asks file to be rotated if it is too big.
 func (s *Appender) Append(line string) {
 	if s.bytesWritten >= *maxFileSize {
 		s.rotateFile()
 	}
 	n, _ := s.writer.WriteString(line)
```
Comment (Woellchen) [inline main.go]: i wonder why this is `line` and not `line + "\n"`. this way you can also save `s.writer.WriteByte('\n')` and only need `s.bytesWritten += n`

File: main.go [hunk 3]
```diff
@@ -167,12 +192,20 @@ func (s *Appender) archiveFileName() string {
 	return s.filePath + "_" + ts
 }
 
+// Append inserts line at the end of file and asks file to be rotated if it is too big.
 func (s *Appender) Append(line string) {
```
Comment (Woellchen) [inline main.go:194]: this whole function and many other parts in this file completely ignore errors. i would add error handling if i were you, but that is on you

**Review by Woellchen — CHANGES_REQUESTED** [review]

File: main.go [hunk 4]
```diff
@@ -33,6 +39,10 @@ func main() {
 	}
 	flag.Parse()
 
+	if (*syslogTarget != "" && *syslogRegexp == "") || (*syslogTarget == "" && *syslogRegexp != "") {
```
Comment (Woellchen) [inline main.go]: i dont see a reason to force a regex

Comment (Woellchen) [reply main.go]: anyway how did you come up with this if in the first place? :D

File: main.go (hunk 3 above)
Comment (TuxPowered42) [reply main.go:194]: This will be fixed separately.

**Review by Woellchen — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (TuxPowered42) [reply main.go]: Already fixed.

File: main.go (hunk 2 above)
Comment (TuxPowered42) [reply main.go]: Discussed 2f2.

**Pull request merged**